book/4 - MultiWorlds/3 - Linking Worlds with a Plugin.html

```diff
@@ -0,0 +1,3 @@
+<p>
+	The easiest way to link worlds together is using a dedicated plugin, such as <a href="http://forum.mc-server.org/showthread.php?tid=2157">Portal V2</a>. More plugins can be found at the <a href="http://forum.mc-server.org/forumdisplay.php?fid=2">respository</a>.
```

Comment:
Ok, I will switch the subsection order and describe the built-in as "easiest" while the plugin method as "most cofigurable".